### PR TITLE
Sidebar version 1

### PR DESCRIPTION
**Pull request by matanbroner**
Not implemented:

- Collapsible sidebar
- Proper color scheme

### CODE REVIEW

**Comment by cpepi001** [comment]
So, I tried to run it with `rails server` but I am getting the following error:

p#2

What I did:
 - Clone the repo
 - Checkout to this branch
 - `$rails server`
 - Got an error for `pg-1.4.5` and a hint to run `$bundle install`
 - `$bundle install`
 - Error `In Gemfile: pg`
 - Modified Gemfile according to [this](https://github.com/tamu-edu-students/hw-hello-rails/blob/main/docs/Part4.md)
 - `$rails server`

```
ruby -v && rails -v && bundle -v
ruby 3.2.0 (2022-12-25 revision a528908271) [x86_64-linux]
Rails 7.0.4.2
Bundler version 2.4.6
```

File: Gemfile [hunk 1]
```diff
@@ -77,3 +77,5 @@ gem "rubocop-rails", "~> 2.17"
 gem "pg", "~> 1.4"
```
Comment (cpepi001) [inline Gemfile:77]: Let's move that to a production group

```
group :production do
  gem "pg", "~> 1.4"
end
```

Comment (matanbroner) [reply Gemfile:77]: Good call, will make the change

**Comment by matanbroner** [comment]
@cpepi001 For the sass error, you need to make sure you have SCSS installed locally. My bad for not mentioning that in the PR.

**Review by liamrberney — APPROVED** [review]
Looks good, installing pg was an easy fix.

**Review by cpepi001 — APPROVED** [review]

**Pull request merged**